test/e2e/nstemplatetier_test.go

```diff
@@ -0,0 +1,28 @@
+package e2e
+
+import (
+	"testing"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-e2e/doubles"
+	"github.com/codeready-toolchain/toolchain-e2e/wait"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateOrUpdateNSTemplateTierAtStartup(t *testing.T) {
+	// given
+	tierList := &toolchainv1alpha1.NSTemplateTierList{}
+	ctx, awaitility := doubles.InitializeOperators(t, tierList)
+	defer ctx.Cleanup()
+	hostAwait := wait.NewHostAwaitility(awaitility)
+
+	// check the "advanced" NSTemplateTier exists (just created)
+	err := hostAwait.WaitForNSTemplateTier("advanced")
+	require.NoError(t, err)
+
+	// check the "basic" NSTemplateTier exists, and all its Namespace revisions are different from `000000a`,
+	// which is the value specified in the initial manifest
+	err = hostAwait.WaitForNSTemplateTier("basic", wait.NSTemplateTierSpecHaving(wait.Not(wait.NamespaceRevisions("000000a"))))
```

Comment:
I like this one :+1:

Reply:
thanks ;)

Reply:
I had the same thinking that the dot import would improve the readability but I didn't want to bother you with additional changes inside of this PR. 
Anyway, +1 from my side for using dot imports

Reply:
ok, let me do it ;)

Reply:
done in f36a7bc